### PR TITLE
Apply CNR comments.

### PR DESCRIPTION
**Pull request by pvretano**

### CODE REVIEW

File: core/clause_6_introduction.adoc [hunk 1]
```diff
@@ -2,33 +2,37 @@
...
 
-* WGS 84 is the only coordinate reference system that is supported;
+* WGS 84 is the only allowed Coordinate Reference System (CRS);
+* No support for metadata so such properties as the provenance of the data cannot be specified;
+* No styling or symbology information is specified;
 * Geometries are restricted to points, curves and surfaces with linear interpolation; and
 * No general capability is available to specify the schema of a feature, its type or its temporal characteristics.
 
+NOTE: PAV Comment.  I added Carl's comments about provenance metadata and styling information but I am not sure they are relevant here.  There is nothing preventing someone from adding additional members to GeoJSON to support these requirements.  They would simply be ignored by GeoJSON clients.  Also, I think these requirements are out-of-scope for the JSON-FG SWG.  The Metadata DWG (?) and the Styling/Symbology SWG (?) would need to consider extending GeoJSON with additional members to accommodate these requirements.
```
Comment (cportele) [inline core/clause_6_introduction.adoc:13]: Meeting 2022-03-07: Yes, these use cases are out-of-scope for JSON-FG and we should not add them in this document. We will limit the list to the topics that JSON-FG addresses. The key point is that we are not prohibiting others to add such information to a GeoJSON/JSON-FG document. @pvretano will remove the two bullets again.

File: core/clause_5_conventions.adoc [hunk 2]
```diff
@@ -9,16 +9,22 @@ All requirements and conformance tests that appear in this document are denoted
...
 
-The JSON schemas have been designed to support extensions. JSON-FG documents that include extensions will still validate against the JSON-FG schemas.
+The JSON schemas are designed to support extensions. JSON-FG documents that include extensions will still validate against the JSON-FG schemas.
 
 The "$id" member of the schemas does not include information about the version of the Standard or the part of the JSON-FG standard. Future revisions of this Standard or additional parts of JSON-FG will update and extend the JSON-FG schemas. Any change will not invalidate existing JSON-FG documents.
 
-<add an Annex with examples how future parts or communities can extend the schemas>
\ No newline at end of file
+NOTE: CNR Comment about this last paragraph: Is some clarification needed? From JSON schema: It is also best practice to include an $id property as a unique identifier for each schema. For now, just set it to a URL at a domain you control, for example:
```
Comment (cportele) [inline core/clause_5_conventions.adoc:24]: Meeting 2022-03-07: Clarify that "$id" is specified in JSON Schema.

File: core/clause_6_introduction.adoc [hunk 3]
```diff
@@ -2,33 +2,37 @@
...
+In the first scenario (API access), the client will typically request the JSON-FG representation of the feature using an HTTP header like `Accept: application/vnd.ogc.fg+json, application/geo+json;q=0.8` and a `crs=http://www.opengis.net/def/crs/EPSG/0/5555` query parameter. The header states that the client prefers JSON-FG, but if JONS-FG is not available will also accept GeoJSON. The response will include the headers `Content-Type: application/vnd.ogc.fg+json` as well as `Content-Crs: \http://www.opengis.net/def/crs/EPSG/0/5555` and include the JSON-FG building blocks including the geometry in the requested projected CRS and temporal information.
```
Comment (cportele) [inline core/clause_6_introduction.adoc:36]: JONS -> JSON

File: core/clause_7_building_blocks.adoc [hunk 4]
```diff
@@ -97,11 +97,13 @@ NOTE: This Standard only provides guidance how to represent feature data in JSON
 
 This describes the initial range of instant values. This range may be extended in the future to support additional use cases. Clients processing instant values must be prepared to receive other values. Clients may ignore values that they do not understand.
 
+NOTE: CNR Comment: What is "This"?  This clause?  This(these) example(s)?
```
Comment (cportele) [inline core/clause_7_building_blocks.adoc:100]: Meeting 2022-03-07: @cportele will resolve this.

File: core/clause_7_building_blocks.adoc [hunk 5]
```diff
@@ -132,18 +134,20 @@ Unbounded intervals ends are represented by a double-dot string ("..") for the s
 
 This describes the initial range of interval values. This range may be extended in the future to support additional use cases. Clients processing interval values must be prepared to receive other values. Clients may ignore values that they do not understand.
 
+NOTE: CNR Comment: What is "This"?  This clause?  This(these) example(s)?
```
Comment (cportele) [inline core/clause_7_building_blocks.adoc:137]: Meeting 2022-03-07: Same as above - @cportele will fix this.

**Review by cportele — CHANGES_REQUESTED** [review]
Meeting 2022-03-07: Merge as soon as the comments have been addressed.

File: core/clause_7_building_blocks.adoc (hunk 4 above)
Comment (cportele) [reply core/clause_7_building_blocks.adoc:100]: @pvretano - change the paragraph to:

"Dates and timestamps are the initial range of instant values. The range may be extended in the future to support additional use cases. Clients processing values of `time` must be prepared to receive other values. Clients may ignore values that they do not understand."

**Comment by pvretano** [comment]
02-MAY-2022: @pvretano to address last few remaning comments from @cportele and then just go ahead and merge.

File: core/clause_6_introduction.adoc (hunk 1 above)
Comment (cportele) [reply core/clause_6_introduction.adoc]: Resolved by 97ebaf7

File: core/clause_6_introduction.adoc (hunk 3 above)
Comment (cportele) [reply core/clause_6_introduction.adoc]: Resolved by 790ef68

File: core/clause_7_building_blocks.adoc (hunk 4 above)
Comment (cportele) [reply core/clause_7_building_blocks.adoc]: Resolved by 2f1bd69

File: core/clause_7_building_blocks.adoc (hunk 5 above)
Comment (cportele) [reply core/clause_7_building_blocks.adoc]: Resolved by 2f1bd69

File: core/clause_5_conventions.adoc (hunk 2 above)
Comment (cportele) [reply core/clause_5_conventions.adoc]: Resolved by ae4a5ae

**Pull request merged**